docs/writing-docs/docs-page.md

```diff
@@ -1,205 +1,188 @@
 ---
-title: 'DocsPage'
+title: 'Docs'
 ---
 
-When you install [Storybook Docs](https://storybook.js.org/addons/@storybook/addon-docs), DocsPage is the zero-config default documentation that all stories get out of the box. It aggregates your [stories](../get-started/whats-a-story.md), text descriptions, docgen comments, [args tables](./doc-block-argstable.md), and code examples into a single page for each component.
+Documentation is an essential part of modern UI development, as it serves as a reference for developers who may need to understand or integrate your components into their own projects. Well-written documentation can help to promote the project and make it more accessible to a broader audience. Overall, the importance of writing documentation cannot be overstated, as it is a key factor in the success of any project.
 
-The best practice for docs is for each component to have its own set of documentation and stories.
+In Storybook, this workflow happens automatically. When you write stories for your components, you create living, interactive, sustainable documentation.
 
-## Component parameter
+## How does documentation work in Storybook?
 
-Storybook uses the `component` key in the story file’s default export to extract the component's description and props.
+You start by writing a [**story**](../writing-stories/introduction) for your component to define the different states it can be in. Then generate the baseline **documentation** for your story via `tags`. Finally, customize the documentation with [MDX](./mdx.md) and Storybook's **Doc Blocks** to fully control how the content gets rendered.
```

Comment:
I think we should make it clear that the final step is optional, and isn't required.